metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -838,12 +838,13 @@ class MetalsLanguageServer(
           if (userConfig.symbolPrefixes != old.symbolPrefixes) {
             compilers.restartAll()
           }
-          if (userConfig.bloopVersion != old.bloopVersion) {
+          val expectedBloopVersion = userConfig.currentBloopVersion
+          val correctVersionRunning =
+            buildServer.map(_.version).contains(expectedBloopVersion)
```

Comment:
Will this trigger a warning if I'm running a custom version of Bloop locally? Ideally, it would be good to only trigger this when the user has explicitly configured a bloop version through the settings before (I personally don't change this setting).

Reply:
Fixed!